orttraining/orttraining/test/python/orttraining_test_ortmodule_autograd.py

```diff
@@ -0,0 +1,881 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Import external libraries.
+import copy
+import os
+import pytest
+import threading
+import torch
+from torch.nn.parameter import Parameter
+
+# Import ORT modules.
+import _test_helpers
+from onnxruntime.training.ortmodule import ORTModule
+from onnxruntime.training.ortmodule._graph_execution_manager_factory import GraphExecutionManagerFactory 
+
+torch.manual_seed(1)
+
```

Comment:
should you also use `onnxruntime.set_seed()` API?

Reply:
[wehci]